plugins/UM3NetworkPrinting/resources/qml/MonitorPrinterCard.qml

```diff
@@ -94,15 +94,15 @@ Item
                 {
                     id: printerNameLabel
                     // color: "#414054" // TODO: Theme!
```

Comment:
Is this line needed anymore?

Reply:
Good catch this one can be removed. Let me double check for comments.

Reply:
Fixed